src/main/resources/templates/courts.html

```diff
@@ -7,15 +7,16 @@
 <body class="dark-edition" style="background-image: url(/img/upcoming_background.jpg); background-size: cover">
 <div class="wrapper">
 
-   <!--Side Bar-->
-   <div th:insert="fragments/sidebar :: sidebar"></div>
-   <!--End Side Bar-->
+    <!--Side Bar-->
```

Comment:
Ten cały plik jest do wywalenia, te zmiany to jest revert poprawek po merge'owaniu

Reply:
Można chyba łatwo wywalić tu z poziomu GitHuba ;) "..." obok Viewed

Reply:
Też to widziałem, faktycznie można.

Reply:
Ogólnie testowałem opcję Delete File na innym branchu i niestety nie usuwa ona zmian tylko commituje usunięcie całego pliku, więc raczej się nie przyda. Cherry Pick też nie pomoże, bo w commicie z tą zmianą są inne zmiany, więc pewnie będzie trzeba po merge'u przywrócić stan z mastera.